### ISSUE TITLE
GPT-4 Prompts used in scripts/notebooks/gpt4_inference.ipynb

### ISSUE THREAD

**Issue by araag2** [EdinburghClinicalNLP/semeval_nli4ct#16]
Hello, I was wondering if it would be possible to provide the GPT-4 Prompts used in this notebook file (seem to be loading from"/Users/aryopg/Downloads/test_prediction_df.table (1).json"), in order to more accurately be able to reproduce the reported results.

If they're available elsewhere and I'm missing them, I apologize for the bother.

**Comment by aryopg** [comment]
Ah apologies for forgetting to change that and for the late reply (I was on an extended break). That file is basically a prediction file generated by the other models that are stored in WandB. The reason why we used that is to skip the [preprocessing step](https://github.com/EdinburghClinicalNLP/semeval_nli4ct/blob/main/src/datasets/chat_dataset.py) and just get the input. An example of this file can be found [here](https://github.com/user-attachments/files/17492467/test_prediction_df.table.1.json).

In conclusion, you can also not use that file and go through the same process in the [`src/trainer.py`](https://github.com/EdinburghClinicalNLP/semeval_nli4ct/blob/main/src/trainer.py).

**Comment by araag2** [comment]
Thanks for the reply!
